routes/auth.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use App\Http\Controllers\Auth\AuthenticatedSessionController;
+use App\Http\Controllers\Auth\EmailVerificationNotificationController;
+use App\Http\Controllers\Auth\NewPasswordController;
+use App\Http\Controllers\Auth\PasswordResetLinkController;
+use App\Http\Controllers\Auth\RegisteredUserController;
+use App\Http\Controllers\Auth\VerifyEmailController;
+use Illuminate\Support\Facades\Route;
+
+Route::post('/register', [RegisteredUserController::class, 'store'])
+    ->middleware('guest')
```

Comment:
Consider to use https://laravel.com/docs/11.x/routing#route-group-middleware here